flipperdevices/flipperzero-firmware
# [FL-3081] fbt: removed DEBUG_TOOLS & FAP_EXAMPLES flags

# What's new

- [FL-3081] fbt: deprecated DEBUG_TOOLS & FAP_EXAMPLES flags
- fixed PYTHONHOME leaking from user's environment into fbt on *nix

# Verification 

- run fbt flash_usb_full, check that no debug/example apps are deployed on SD card
- run `fbt fap_dist`, check that all known apps are deployed to `dist` folder

# Checklist (For Reviewer)



[FL-3081]: https://flipperzero.atlassian.net/browse/FL-3081?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
